doc/releases/nautilus.rst

```diff
@@ -1,3 +1,173 @@
+v14.2.17 Nautilus
+=================
+
+This is the 17th backport release in the Nautilus series. We recommend
+users to update to this release.
+
+Notable Changes
+---------------
+
+* $pid expansion in config paths like `admin_socket` will now properly expand
+  to the daemon pid for commands like `ceph-mds` or `ceph-osd`. Previously only
+  `ceph-fuse`/`rbd-nbd` expanded `$pid` with the actual daemon pid.
+
```

Comment:
Can we please include the following:
```
RADOS

* PG removal has been optimized in this release.
* Memory allocations are tracked in finer detail in BlueStore and displayed as a part of the ``dump_mempools`` command.
* Several fixes in BlueStore to improve stability.
```

Reply:
Is the formatting (`RADOS:` for each bullet point) okay?  I didn't see a sub-header anywhere like you suggested.

Reply:
yeah